app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -92,6 +93,61 @@ def test_serializer_outputs_expected_schema_for_live_experiment(self):
                 "owner": "owner@example.com",
                 "slug": "rapid-experiment",
                 "status": Experiment.STATUS_LIVE,
+                "reject_feedback": None,
+            },
+        )
+
+    def test_serializer_outputs_expected_schema_for_rejected_experiment(self):
+        owner = UserFactory(email="owner@example.com")
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_ACCEPTED,
+            type=Experiment.TYPE_RAPID,
+            rapid_type=Experiment.RAPID_AA_CFR,
+            owner=owner,
+            name="rapid experiment",
+            slug="rapid-experiment",
+            objectives="gotta go fast",
+            audience="all_english",
+            features=["picture_in_picture"],
```

Comment:
Dumb[er than usual] question: Should we be pulling the `audience` and `features` from constants in the nimbus_shared package? Not sure if those could subtly change upstream in the future and break these tests/validations. Or maybe not, what do I know...